packages/react-router/lib/server-runtime/serverHandoff.ts

```diff
@@ -1,29 +1,19 @@
-import type { HydrationState } from "../router/router";
 import type { CriticalCss, FutureConfig } from "../dom/ssr/entry";
 import { escapeHtml } from "./markup";
+import type { ServerBuild } from "./build";
 
-type ValidateShape<T, Shape> =
-  // If it extends T
-  T extends Shape
-    ? // and there are no leftover props after removing the base
-      Exclude<keyof T, keyof Shape> extends never
-      ? // we are good
-        T
-      : // otherwise it's either too many or too few props
-        never
-    : never;
```

Comment:
No longer needed because `state` is never included in the handoff string now - that was leftover from pre-turbo-stream